config/ldes-delta-pusher/ldes-instances.ts

```diff
@@ -0,0 +1,232 @@
+export const ldesInstances = {
+  public: {
```

Comment:
config in one place only!

Reply:
well except for special cases mandataris and besluit, that would have taken it a little too far